bin/propolis-standalone/src/main.rs

```diff
@@ -876,31 +879,32 @@ fn generate_smbios(params: SmbiosParams) -> anyhow::Result<smbios::TableBytes> {
 
     let smb_type4 = smbios::table::Type4 {
         // central processor
-        proc_type: 0x03,
+        proc_type: type4::ProcType::Cpu,
         proc_family,
         proc_manufacturer,
         proc_id,
         proc_version: proc_version.as_str().try_into().unwrap_or_default(),
         // cpu enabled, socket populated
-        status: 0x41,
+        status: type4::ProcStatus::Enabled,
         // unknown
         proc_upgrade: 0x2,
         // make core and thread counts equal for now
         core_count: params.num_cpus,
         core_enabled: params.num_cpus,
         thread_count: params.num_cpus,
         // 64-bit capable, multicore
-        proc_characteristics: 0xc,
+        proc_characteristics: type4::Characteristics::IS_64_BIT
+            | type4::Characteristics::MULTI_CORE,
         ..Default::default()
     };
 
     let mut smb_type16 = smbios::table::Type16 {
         // system board
-        location: 0x3,
+        location: type16::Location::SystemBoard,
         // system memory
-        array_use: 0x3,
+        array_use: type16::ArrayUse::System,
         // unknown
-        error_correction: 0x2,
+        error_correction: type16::ErrorCorrection::Unknown,
```

Comment:
Omitted for standalone, but not here?

Reply:
ah, the default impl populates this with `Unknown` now, and I went back and
forth on whether I wanted to use the default impl for `Unknown` fields when we
don't default to unknown for stuff that isn't an enum/bitflags yet. I'll go back
and add it in standalone.